html-template/index.html

```diff
@@ -4,11 +4,6 @@
   <meta charset="utf-8">
   <meta name="theme-color" content="#000000">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <!--
```

Comment:
manifest was causing 401s to auth all the time, so removed as not needed (only used for android home screen apparently anyway and out of the box react that should be removed)